assertk-coroutines/src/commonMain/kotlin/assertk/coroutines/assertions/any.kt

```diff
@@ -10,8 +10,16 @@ import assertk.assertions.support.appendName
  * @param extract The suspend function to extract the property value out of the value of the current assert.
  *
  * ```
- * assertThat(person).suspendCall("name()", { it.name() }).isEqualTo("Sue")
+ * assertThat(person).having("name()", { it.name() }).isEqualTo("Sue")
  * ```
  */
+suspend fun <T, P> Assert<T>.having(name: String, extract: suspend (T) -> P): Assert<P> =
```

Comment:
If we make the other one `inline`, we can eliminate this overload. Do we want that? Both APIs are public.

Reply:
Yeah doing that would be fine